CONTRIBUTING.md

```diff
@@ -55,9 +55,12 @@ conda install pytorch -c pytorch-nightly
 pip install cmake ninja
 ```
 
+### Install other requirements
+
```

Comment:
the content under this title belongs to the previous section.
```suggestion
```

Reply:
I've removed it. Thanks :)